### ISSUE TITLE
Spacemouse WiFi not connected

### ISSUE THREAD

**Issue by lastnio** [smartavionics/RawMouse#19]
Hi there,
i can't get the 3dconnexion pro wifi to connect with cura 4.13.0,

I use macbook pro, macOS Monterey 12.2.

can someone help me

Thank you.

**Comment by smartavionics** [comment]
Could you please provide the cura.log as that may provide some hint as to what is going wrong. Thanks.

**Comment by smartavionics** [comment]
ah, I just realised that you are using a version of macos that probably won't ever work with RawMouse due the the fact that the 3dconnexion driver requires exclusive access to the mouse. See https://github.com/smartavionics/RawMouse/issues/9

**Comment by lastnio** [comment]
Hi there,
maybe you can brave look at the logfile

Thank you.
[cura.log](https://github.com/smartavionics/RawMouse/files/8008924/cura.log)

**Comment by smartavionics** [comment]
Yeah, it's the same issue as occurs on Big Sur, the 3dconnexion driver opens the device in exclusive mode and so software (like RawMouse) that doesn't use the 3dconnexion driver cannot access the device. Before Big Sur came along, the 3dconnexion driver didn't open the device exclusively. Sorry, there's nothing I can do about this.

```
2022-01-30 14:43:57,847 - DEBUG - [RawMouse] RawMouse.RawMouse._run [167]: Trying to open IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS06@14100000/USB2.0 Hub             @14100000/AppleUSB20Hub@14100000/AppleUSB20HubPort@14110000/IOUSBHostDevice@14110000/AppleUSB20Hub@14110000/AppleUSB20HubPort@14112000/SpaceMouse Pro Wireless Receiver@14112000/IOUSBHostInterface@0/AppleUserUSBHostHIDDevice
2022-01-30 14:43:57,848 - ERROR - [RawMouse] RawMouse.RawMouse._run [202]: Exception while reading HID events: open failed
```

**Comment by lastnio** [comment]
ok i understand, bat if I use SuperSlycer, then the 3dconnexion works
